test/quantization/pt2e/test_x86inductor_quantizer.py

```diff
@@ -965,3 +981,86 @@ def test_qat_conv2d_unary_with_quantizer_api(self):
                     node_list,
                     is_qat=True,
                 )
+
+    @skipIfNoX86
+    def test_qat_conv2d_binary_with_quantizer_api(self):
```

Comment:
nit: we can remove `_with_quantizer_api` now

Reply:
Thanks and removed.